### PR TITLE
Add remove reminder functionality to close #9

### LINKED ISSUE #9: Delete existing reminder
User can remove/delete a reminder.

* Users should be able to remove a reminder from either the list of reminders or from an individual reminder’s page.
* If they do it from the individual reminder’s page, then the application should reroute them back to the `/reminders` route.

* **Bonus** Implement a flash message that tells the user which reminder was deleted. AKA ` "Don't forget to pick up your dry cleaning!" Successfully Deleted! `

### PR DESCRIPTION
**Pull request by Mickyfen17**
## Purpose

To add delete functionality for each reminder from main screen and within each individual reminder

## Approach

Add remove reminder actions and call the action on each remove button clic

### Test coverage 

Test for reminder content length before and after a remove reminder click on main page.
Test for reminder content length, check for reminder article length upon click on reminder title &
test reminder content length after click on remove reminder button within article.

### CODE REVIEW

**Comment by Mickyfen17** [comment]
@Tman22 @martensonbj

File: app/controllers/reminders.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    removeReminder(reminder) {
+      reminder.deleteRecord();
+      reminder.save();
```
Comment (Tman22) [inline app/controllers/reminders.js]: Look into 'destroyRecord()'

File: app/controllers/reminders/reminder/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    removeReminder(model) {
```
Comment (Tman22) [inline app/controllers/reminders/reminder/index.js:5]: Semi-colons

File: tests/acceptance/reminder-list-test.js [hunk 3]
```diff
@@ -79,3 +79,42 @@ test('edit reminders', function(assert) {
     assert.equal(Ember.$('.reminder-body').text(), 'New reminder');
   });
 })
+
+test('delete reminder from main page', function(assert) {
+  server.createList('reminder', 5);
+
+  visit('/reminders');
+  andThen(function() {
```
Comment (Tman22) [inline tests/acceptance/reminder-list-test.js]: Fix this spacing. I like these tests though.

**Comment by Mickyfen17** [comment]
@Tman22 @martensonbj 
Fixes carried out

File: app/controllers/reminders.js (hunk 1 above)
Comment (Mickyfen17) [reply app/controllers/reminders.js]: destroyRecord deletes and saves therefore no need to delete and then save on line below. Thanks for the tip. @Tman22

File: app/controllers/reminders/reminder/index.js (hunk 2 above)
Comment (Tman22) [reply app/controllers/reminders/reminder/index.js:5]: You should fix semi-colons still and use 'destroyRecord' here too.

**Comment by Mickyfen17** [comment]
@Tman22 
Fixed

**Comment by martensonbj** [comment]
Appreciate the work on updating the semi-colon consistency 👍

**Pull request merged**